src/components/BooksContainer.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable arrow-body-style */
+import styles from './BooksContainer.module.css';
+import Book from './Book';
+
+const Books = () => {
+  return (
+    <>
+      <div className={styles.innerContent}>
+        <div>
+          <ul>
+            <Book />
```

Comment:
And here we need to pass the props for our book. In the future, if we will have multiple books from an array, we will need a loop for this. But for now, it will do only with one book.
It should look something like this in the case of only one book:

![image](https://user-images.githubusercontent.com/83596135/165977944-79029b98-8687-4e1f-a19a-d2ebc482587b.png)


if you want to see how it will look if you will have multiple books (you can use it in your project if you want):
![image](https://user-images.githubusercontent.com/83596135/165978735-7b507d0e-1951-4257-ad2f-61a9c6025cc2.png)


Note: for the moment don't worry about deactivating linters; we will learn about keys later on;

Reply:
Thanks for the helpful explainer @nicupop729 .